src/index.html

```diff
@@ -8,6 +8,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star__active"></div>
```

Comment:
<img width="876" alt="Screenshot 2023-09-18 at 17 20 31" src="https://github.com/mate-academy/layout_stars/assets/53228259/043f5aaa-83ca-4aba-965c-0fcb8a14fe05">

delete __active and use modifier for adding active star